src/control/cmd/daos/container.go

```diff
@@ -87,7 +87,7 @@ func (cmd *containerBaseCmd) openContainer() error {
 		defer freeString(cLabel)
 
 		cmd.log.Debugf("opening container: %s", cmd.contLabel)
-		rc = C.daos_cont_open(cmd.cPoolHandle, cLabel, openFlags,
+		rc = C.daos_cont_open2(cmd.cPoolHandle, cLabel, openFlags,
```

Comment:
i am not sure we should be calling daos_cont_open2 in this case. daos_cont_open should work fine especially if this is a label.

Reply:
Honestly, I'm not sure how it would ever work

113 int
114 daos_cont_open(daos_handle_t poh, const char *cont, unsigned int flags,
115                daos_handle_t *coh, daos_cont_info_t *info, daos_event_t *ev)
116 {
117         daos_cont_open_t        *args;
118         tse_task_t              *task;
119         const unsigned char     *uuid = (const unsigned char *) cont;
120         int                      rc;
121 
122         DAOS_API_ARG_ASSERT(*args, CONT_OPEN);
123         if (!daos_uuid_valid(uuid))
124                 return -DER_INVAL;

The old API presents as a const char * but assumes it's a uuid